### PR TITLE
Add new version of coredns

### PR DESCRIPTION
**Pull request by pipo02mix**
New release of coredns [1.5.1](https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md)

### CODE REVIEW

**Comment by pipo02mix** [comment]
tested succesfully on ginger

**Comment by MarcelMue** [comment]
The port change will 100% lead to issues on installations such as `asgard` - there are already network policies involved and we ran into trouble during the last upgrade there.

**Comment by pipo02mix** [comment]
I am adding a warning in the changelogs, but do we have a place where I can link to explain why and how to solve it?

**Comment by MarcelMue** [comment]
> I am adding a warning in the changelogs, but do we have a place where I can link to explain why and how to solve it?

Maybe it makes sense to have a public gist somewhere? That is easy to list and access IMO

File: service/controller/aws/v16/version_bundle.go [hunk 1]
```diff
@@ -32,6 +32,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Switched security context to non-root user.",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "coredns",
+				Description: "Default container port changed from 53 to 1053. Please read project's changelog carefully https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md#v050",
```
Comment (kopiczko) [inline service/controller/aws/v16/version_bundle.go]:
```suggestion
				Description: "Change default container port from 53 to 1053. Please read project's changelog carefully https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md#v050",
```

File: service/controller/aws/v17/version_bundle.go [hunk 2]
```diff
@@ -7,6 +7,11 @@ import (
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "coredns",
+				Description: "Updated to 1.5.1. More info here: https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md",
```
Comment (kopiczko) [inline service/controller/aws/v17/version_bundle.go]:
```suggestion
				Description: "Update to 1.5.1. More info here: https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md",
```

File: service/controller/azure/v16/version_bundle.go [hunk 3]
```diff
@@ -32,6 +32,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Switched security context to non-root user.",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "coredns",
+				Description: "Default container port changed from 53 to 1053. Please read project's changelog carefully https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md#v050",
```
Comment (kopiczko) [inline service/controller/azure/v16/version_bundle.go]:
```suggestion
				Description: "Change default container port from 53 to 1053. Please read project's changelog carefully https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md#v050",
```

File: service/controller/azure/v17/version_bundle.go [hunk 4]
```diff
@@ -7,6 +7,11 @@ import (
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "coredns",
+				Description: "Updated to 1.5.1. More info here: https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md",
```
Comment (kopiczko) [inline service/controller/azure/v17/version_bundle.go]:
```suggestion
				Description: "Update to 1.5.1. More info here: https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md",
```

File: service/controller/kvm/v17/version_bundle.go [hunk 5]
```diff
@@ -7,6 +7,11 @@ import (
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "coredns",
+				Description: "Updated to 1.5.1. More info here: https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md",
```
Comment (kopiczko) [inline service/controller/kvm/v17/version_bundle.go]:
```suggestion
				Description: "Update to 1.5.1. More info here: https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md",
```

File: service/controller/kvm/v16/version_bundle.go [hunk 6]
```diff
@@ -32,6 +32,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Switched security context to non-root user.",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "coredns",
+				Description: "Default container port changed from 53 to 1053. Please read project's changelog carefully https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md#v050",
```
Comment (kopiczko) [inline service/controller/kvm/v16/version_bundle.go]:
```suggestion
				Description: "Change default container port from 53 to 1053. Please read project's changelog carefully https://github.com/giantswarm/kubernetes-coredns/blob/master/CHANGELOG.md#v050",
```

**Comment by pipo02mix** [comment]
CI green, added extra changelog and notified SE, merging

**Pull request merged**